src/extensions/extra/ImageUrlCodeHints/camera/selfieWidget.html

```diff
@@ -0,0 +1,15 @@
+<div id="selfie-widget">
+    <div id="selfie-liveView">
+        <div id="selfie-video-bg"></div>
+            <video id="selfie-video"></video>
```

Comment:
Don't indent things that aren't children.  This should be at the same indent level as the div above, since you closed it.